pyproject.toml

```diff
@@ -1,3 +1,76 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "trio"
+description = "A friendly Python library for async concurrency and I/O"
+authors = [{name = "Nathaniel J. Smith", email = "njs@pobox.com"}]
```

Comment:
Maybe `maintainers` should be filled in too?

Reply:
that's probably quite a long list, and I want to keep the metadata identical in this PR

Reply:
Could write something like "project collaborators" as a single entry. But you're right. It's out of the scope.

Reply:
@agronholm @webknjaz https://github.com/python-trio/trio/issues/2868